controllers/errors.controller.js

```diff
@@ -0,0 +1,17 @@
+// const manageCustomError = (err,req,res,next) => {
+//     if(err.msg && err.status){
+//         res.staus(err.status).send({msg: err.msg})
+//     }else{
+//         next(err)
+//     }
+// };
```

Comment:
I shouldn't see commented out code as part of a commit which is meant to be production-ready, please remove this